### ISSUE TITLE
Edit workflow confusion?

### ISSUE THREAD

**Issue by nasso** [steveklabnik/jujutsu-tutorial#19]
https://github.com/steveklabnik/jujutsu-tutorial/blob/main/src%2Freal-world-workflows%2Fthe-edit-workflow.md#L54-L58

here you mention how we may want to "refactor printing out into its own function"

### expected behaviour

the rest of the example goes through the steps of how we would do that refactoring

### actual behaviour

we just add a comment

### additional context

operating system: android
browser: firefox focus
theme: navy
rocket league rank: c1d1
effort put in writing this critical bug report: yes

(maybe you just didn't want to talk about conflicts at this time?)

**Comment by steveklabnik** [comment]
> (maybe you just didn't want to talk about conflicts at this time?)

Yeah, so originally I had some stuff with conflicts earlier than this, and then later, came back and moved that stuff to a later section. So this is an inconsistency.
